DeepL.gs

```diff
@@ -75,6 +82,22 @@ function DeepLTranslate(input, sourceLang, targetLang, glossaryId) {
     if (glossaryId) {
         formData['glossary_id'] = glossaryId;
     }
+    if (options) {
+        if (!Array.isArray(options) ||
+            !Object.values(options).every(function(value) {
```

Comment:
I don't think this condition is completely correct.
`!Array.isArray(options) || ...` means we throw if `options` is not an array (`[...]`), and then `... || !Object.values.every....` checks that each entry is of length two. I assume this is to support the cell range approach? But I would expect:

- This to throw when we pass a dictionary/an Object with the options (`Arrays.isArray({'tag_handling':'xml'})` is false)
- This to break once we are beyond row 10/column 26 (Cell A10 has length 3, right?)

Reply:
The `{..}` syntax used in Google Sheets is deceptive -- in Apps Script it always seems to be converted to an array of arrays. It is possibly a Google-defined type (maybe CellRange or something) that works like an array.

- Passing `{'tag_handling', 'xml'}` as the argument in the cell formula will still yield an array in the script. 
- Passing `{'tag_handling': 'xml'}` didn't seem to work in my tests; in the script it was passed a string with a value like "#NAME".

The arrays' lengths should correspond to the number of columns (i.e. A, B are two columns). I don't think this changes one multi-character row- or column-encodings are used, but I'll check.

Reply:
Got it, that makes sense.

Reply:
Confirmed it works beyond column AA and row 10.